esp/bindings/http/platform/httpservice.cpp

```diff
@@ -1080,7 +1080,7 @@ EspAuthState CEspHttpServer::handleUserNameOnlyMode(EspAuthRequest& authReq)
     if (isEmptyString(userNameIn))
     {
         //Display a GetUserName (similar to login) page to get a user name.
-        askUserLogin(authReq);
+        askUserLogin(authReq, "Empty username");
```

Comment:
Inconsistent punctuation, sometimes you end with period ("User Authentication failed.")